test/test_ops.py

```diff
@@ -443,6 +443,16 @@ def test_noncontiguous_samples(self, device, dtype, op):
                 sample_input.args,
                 sample_input.kwargs,
             )
+
+            # This test requires that the input be a tensor or sequence of 1+ tensors
+            if not isinstance(t_inp, torch.Tensor):
+                if isinstance(t_inp, Sequence):
+                    if not len(t_inp) > 0 or not isinstance(t_inp[0], torch.Tensor):
+                        continue
+                else:
+                    continue
```

Comment:
This is great! I was thinking of doing exactly the same thing with the other tests that fail when the input has no tensors, like
```
               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_view'),
               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_conj_view'),
               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_conj_view'),
```
Could you add add similar logic to these?

Reply:
I actually removed this because there are too many xfails for this test already, so it's going to require it's own change